src/cpp/rtps/attributes/RTPSParticipantAttributes.cpp

```diff
@@ -261,6 +261,7 @@ static void setup_transports_large_datav6(
     {
         Locator_t pdp_locator;
         pdp_locator.kind = LOCATOR_KIND_UDPv6;
+        IPLocator::setIPv6(pdp_locator, "ff02::1");
```

Comment:
Should this be the one we use [here](https://github.com/eProsima/Fast-DDS/blob/17648aa14640c9d2d7041dcb634078b073cc8273/src/cpp/rtps/transport/UDPv6Transport.cpp#L166)?